### ISSUE TITLE
Docs/FAQ update

### ISSUE THREAD

**Issue by perpetual-hydrofoil** [philberty/gccpy#7]
Just an outstanding tool. It would be great to know upfront how feasible compiling things like system libraries,  (simple)json, socket libraries, etc will go. If it's possible to completely compile a REST/http server, for instance, that would be absolutely outstanding especially when looking at all of the options out there (psyco, etc)

Also whether there are any differences between Python 2.x and 3.x when using the tool.

**Comment by philberty** [comment]
Yeah this is what i would love to show currently the goal for gccpy is to get to implement python 2.4 to almost fully correct. With this realse i am pushing to do in a few weeks is a HUGE milestone for the project. Its been in development for so long because implementing all this from scratch is such a huge job but so much of the core has been figured out and implemented now i feel maybe 2 or 3 small milestones after that release you should be able to start compiling full python libraries within reason.

Trying to use numba or any of those fancy native python modules _will_ fail as i have no API/ABI against libpython.so same as PyPy basically.

I really hope to show what your saying about a year from now. I cant describe how hard and time consuming its been to get to this point kind of like writing a kernel not much to show off until you get all the hard stuff out of the way basically.

I unbelievably appreciate your support of the project when i've had to deal with so much flack from PyPy community and doing a difficult project mostly on my own. Thank you so much!

**Comment by philberty** [comment]
I'm aiming to move all my documentation onto github wiki as its more dynamic and fun and more social on github to host an entire project here than source forge or on gcc wiki.

**Comment by perpetual-hydrofoil** [comment]
Awesome man! That's incredibly exciting. Haters are what they are, but anything that is new and different advances the state of the art, so keep on keeping on. I love the metadata files that the compiler spits out for linking ordering -- really smart stuff.

**Comment by philberty** [comment]
With this mile stone you will be able to start doing things like import sys etc which will be great then the next milestone will be threading and more stdlib stuff then the one after will be a garbage collector so these are more self contained projects. Where as for this milestone its been ah get lists working then realise i did something that broke memory allocation and assignment or something silly etc. So i hope you can see what its been like to get to this point.

**Comment by perpetual-hydrofoil** [comment]
I've been writing small things in C for a few decades but this is really such a huge project.. very audacious -- kudos for taking on such a huge challenge.

**Comment by perpetual-hydrofoil** [comment]
I'm keeping an eye on this project. pypy and psyco are not it for me... this has the potential to be really, really helpful for some of us! Keep up the great work!
